tasks/install.yml

```diff
@@ -2,7 +2,7 @@
 
 - name: Install required packages
   become: yes
-  apt:
+  package:
```

Comment:
yeah, I know. But is there any reason why should someone still uses Ansible < 2 that was released more than 6 months ago.